extensions/stackdriver/stackdriver.h

```diff
@@ -58,8 +58,8 @@ class StackdriverRootContext : public RootContext {
   void onStart(std::unique_ptr<WasmData>) override;
   void onTick() override;
 
-  // Get reporter kind of this filter from plugin config.
-  stackdriver::config::v1alpha1::PluginConfig::ReporterKind reporterKind();
+  // Get direction of traffic relative to this proxy.
+  PluginDirection direction();
```

Comment:
is this used other than to check whether or not something is outbound?  could this be `boolean isOutbound()` instead?

Reply:
Ah, good point. Done.